api/location/locationRouter.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express');
+const DB = require('../utils/db-helper');
+const router = express.Router();
+const { requireAdmin } = require('../middleware/authorization');
+
+// GET - View all locations
+router.get('/', (req, res) => {
+  DB.findAll('locations')
+    .then((locations) => {
+      res.status(200).json(locations);
+    })
+    .catch((err) => {
+      res.status(500).json({ error: err.message });
+    });
+});
+
+// GET - View Location by ID
+router.get('/:id', (req, res) => {
+  const { id } = req.params;
+
+  DB.findById('locations', id)
```

Comment:
Has this been tested in Postman or some other way? From what I can tell the db-helper file will end up searching by 'id' and not 'location_id', causing an error.